gammapy/cube/core.py

```diff
@@ -216,13 +216,20 @@ def empty(cls, emin=0.5, emax=100, enumbins=10, eunit='TeV', **kwargs):
             Number of energy bins.
         eunit : str
             Energy unit.
+        mode : {'edges', 'center'}
+            Whether emin and emax denote the bin centers or edges.
         kwargs : dict
             Keyword arguments passed to `~gammapy.image.SkyImage.empty` to create
             the spatial part of the cube.
         """
         image = SkyImage.empty(**kwargs)
-        energy = EnergyBounds.equal_log_spacing(emin, emax, enumbins, eunit)
-        energy_axis = LogEnergyAxis(energy, mode='edges')
+        if mode == 'edges':
+            energy = EnergyBounds.equal_log_spacing(emin, emax, enumbins, eunit)
+            energy_axis = LogEnergyAxis(energy, mode='edges')
+        elif mode == 'center':
+            energy = Energy.equal_log_spacing(emin, emax, enumbins, eunit)
+            energy_axis = LogEnergyAxis(energy, mode='center')
```

Comment:
Please always end with
```
else:
    raise ValueError(...)
```
for multiple-choice based on user string input.
It really helps to get a good error message when you mistype something.